openfisca_uk/variables/finance/benefit/family/pension_credit.py

```diff
@@ -60,32 +67,39 @@ def formula(benunit, period, parameters):
         )
 
 
-class pension_credit_applicable_income(Variable):
+class guarantee_credit_applicable_income(Variable):
     value_type = float
     entity = BenUnit
     label = u"Applicable income for Pension Credit"
     definition_period = YEAR
 
     def formula(benunit, period, parameters):
         INCOME_COMPONENTS = [
+            "personal_benefits",
+            "pension_income",
+            "maintenance_income",
             "employment_income",
             "self_employment_income",
             "property_income",
-            "pension_income",
             "savings_interest_income",
             "dividend_income",
         ]
         income = aggr(benunit, period, INCOME_COMPONENTS)
         tax = aggr(
             benunit,
             period,
-            ["income_tax", "national_insurance"],
+            ["tax"],
         )
-        income += aggr(benunit, period, ["personal_benefits"])
-        income += add(benunit, period, ["child_benefit"])
-        income -= tax
-        income = amount_over(income, 0)
-        return income
+        benefits = add(
+            benunit,
+            period,
+            [
```

Comment:
maybe not in this PR since it's in a lot of places but i find the constant list like `INCOME_COMPONENTS` above more readable than putting it in the `add` functions. I think it'll actually be fewer lines too.

Reply:
Yeah makes sense, especially since the `add(entity, period, [])` pattern seems difficult for `black` to format idiomatically.